.changelog/27573.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+resource/routing_profile: increase the hardcoded queue limit from 10 to 50 to match AWS default quota
```

Comment:
Recommend removing the queue limit since this quota is an adjustable quota. Reference: [Amazon Connect service quotas](https://docs.aws.amazon.com/connect/latest/adminguide/amazon-connect-service-limits.html)

```suggestion
```release-note:bug
resource/aws_connect_routing_profile: removed the hardcoded `queue_configs` limit 
```